akka-actor-typed/src/main/scala/akka/actor/typed/internal/routing/PoolRouterImpl.scala

```diff
@@ -5,11 +5,11 @@
 package akka.actor.typed.internal.routing
 
 import java.util.function
-
 import akka.actor.typed._
 import akka.actor.typed.javadsl.PoolRouter
 import akka.actor.typed.scaladsl.{ AbstractBehavior, ActorContext, Behaviors }
 import akka.annotation.InternalApi
+import akka.japi.Predicate
```

Comment:
We usually only use the akka.japi.function SAMs for things that should be able to throw an exception (which Java stdlib does not) but here there is no reason to allow that, so `java.util.function.Predicate` would be better.